deployment/src/main/java/io/quarkiverse/jackson/jq/deployment/JacksonJqFunctionLoader.java

```diff
@@ -0,0 +1,152 @@
+package io.quarkiverse.jackson.jq.deployment;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import net.thisptr.jackson.jq.BuiltinFunction;
+import net.thisptr.jackson.jq.Function;
+import net.thisptr.jackson.jq.Scope;
+import net.thisptr.jackson.jq.Version;
+import net.thisptr.jackson.jq.VersionRange;
+import net.thisptr.jackson.jq.internal.misc.VersionRangeDeserializer;
+
+/**
+ * Use {@code BuiltinFunctionLoader.getInstance()} to obtain the instance.
+ */
+public class JacksonJqFunctionLoader {
```

Comment:
Is this a copy from https://github.com/eiiches/jackson-jq/blob/develop/1.x/jackson-jq/src/main/java/net/thisptr/jackson/jq/BuiltinFunctionLoader.java#L28?

I've tried to add a few changes on their end but wasn't accepted. Mainly to avoid copying. I think it might be a necessary evil, but I think we should add their license headers or give the credits if we decide to go this route.

Reply:
of course